exercises/book-store/BookStoreTest.fs

```diff
@@ -11,55 +11,59 @@ open BookStore
 let ``Only a single book`` () =
     total [1] |> should equal 8.00
 
-[<Fact(Skip = "Remove to run test")>]
+[<Fact>]
```

Comment:
Why have all the `(Skip = "Remove to run test")` parts been removed? Could you add them back?